app/bundles/EmailBundle/Tests/Transport/MailjetTransportTest.php

```diff
@@ -128,7 +128,7 @@ private function getRequestWithPayload()
     "CustomID": "4-bounce@test.com",
     "Payload": "",
     "blocked": "",
-    "hard_bounce": "",
+    "hard_bounce": "true",
```

Comment:
@escopecz `hard_bounce` will be filled with `true` only when the event is bounced. based on the documentation @kuzmany is current and there is no need for a new test to handle `false` response because it will not occur

Reply:
Well, in the docs the hard_bounce value is type of bool. Not type of string like in this test.

`"hard_bounce": true,` and not `"hard_bounce": "true",`. That's why I asked for a `"false"` value because based on the code it would evaluate to `true` because it's not empty string.

> `hard_bounce` will be filled with `true` only when the event is bounced. based on the documentation

Can you quote the part of the documentation saying it cannot have other value than `true`? I cannot find it.

Reply:
@escopecz if you go to https://dev.mailjet.com/email/guides/webhooks/#bounce-event
You see that you have 1 type of response by event. So if contact is not bounce, the bounce event is not triggered. You can have open event and in this case you don't have the `hard_bounce` property inside.
This property is only for bounce type event, which implies having this value to `true`.

Reply:
@npracht your comment does not react to my concern that we have `string` value in the test and the documentation states it should be `boolean`. That's all I'm saying. The tests in this PR are wrong according to the official Mailjet documentation.